static-assets/components/cstudio-common/common-api.js

```diff
@@ -5696,8 +5696,11 @@ var nodeOpen = false,
         const textarea = document.createElement('textarea');
         const div = document.createElement('div');
         textarea.innerHTML = html;
-        div.innerHTML = textarea.value;
-        return div.innerText.trim() === '';
```

Comment:
Just curious, how come this doesn’t work?

Reply:
This validation was just retrieving the text of the element, not markup like in this case `img`, so if we had a content like
```<p>&nbsp;</p><img src="some/source"/>```
innerText would have returned an empty string.